libcontainer/intelrdt/mbm.go

```diff
@@ -0,0 +1,115 @@
+// +build linux
+
+package intelrdt
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// The flag to indicate if Intel RDT/MBM is enabled
+	isMbmEnabled bool
+
+	enabledMonFeatures monFeatures
+)
+
+type monFeatures struct {
+	mbmTotalBytes bool
+	mbmLocalBytes bool
+	llcOccupancy  bool
+}
+
+// Check if Intel RDT/MBM is enabled
+func IsMbmEnabled() bool {
+	return isMbmEnabled
+}
+
+func getMonFeatures(intelRdtRoot string) (monFeatures, error) {
+	file, err := os.Open(filepath.Join(intelRdtRoot, "info", "L3_MON", "mon_features"))
```

Comment:
Actually, since we're on Linux, you can use `filepath.Join(intelRdtRoot, "info/L3_MON/mon_features")` or even `intelRdtRoot + "/info/L3_MON/mon_features"`.